documents/attribute.go

```diff
@@ -238,7 +238,8 @@ func NewSignedAttribute(keyLabel string, identity identity.DID, account config.A
 	}, nil
 }
 
-func attributeSignaturePayload(did, id, version, value []byte) []byte {
+// AttributeSignaturePayload creates the payload for signing an attribute
+func AttributeSignaturePayload(did, id, version, value []byte) []byte {
```

Comment:
I made the func public but I actually would like to have it as a method of signAttr.

What do you think?

Reply:
Having a function would be preferable. any specific need for a method on AttrSign